src/control/listIndexes.ts

```diff
@@ -1,28 +1,10 @@
 import { IndexOperationsApi } from '../pinecone-generated-ts-fetch';
-
-/**
- * A partial description of indexes in your project.
- *
- * For full information about each index, see
- * { @link Pinecone.describeIndex }
- */
-export type PartialIndexDescription = {
-  /** The name of the index */
-  name: string;
-};
```

Comment:
My assumption is now that the intent is for the API to return the full objects, we don't want to keep the partial type around. Let me know if that's not the case.

Reply:
Yes, exactly. This was just a placeholder until we could return the full object.